app/controllers/businessmatching/updateservice/AddBusinessTypeHelper.scala

```diff
@@ -133,7 +135,7 @@ class AddBusinessTypeHelper @Inject()(val authConnector: AuthConnector,
     val indices = model.responsiblePeople.fold[Set[Int]](Set.empty)(_.index)
 
     OptionT(dataCacheConnector.update[Seq[ResponsiblePerson]](ResponsiblePerson.key) {
-      case Some(people) if model.activity.contains(TrustAndCompanyServices) || model.activity.contains(MoneyServiceBusiness) =>
+      case Some(people) if model.activity.contains(TrustAndCompanyServices) || model.activity.contains(MoneyServiceBusiness) | appConfig.phase2ChangesToggle =>
```

Comment:
Move appConfig toggle call to start of line for readability?

Reply:
Why use single |

Reply:
Done